docs/content/en/docs/getting-started/ports/_index.md

```diff
@@ -72,7 +99,7 @@ On the Admin machine for a Bare Metal provider, the following ports need to be a
 | TCP      | Outbound  | 623        | Rufio IPMI              | All nodes, out-of-band power and next boot ([optional]({{< relref "../baremetal/bare-prereq/#network-requirements" >}})) |
 | TCP      | Outbound  | 80,443     | Rufio Redfish           | All nodes, out-of-band power and next boot ([optional]({{< relref "../baremetal/bare-prereq/#network-requirements" >}})) |
 
-## VMware provider
+##### VMware provider
```

Comment:
```suggestion
### VMware provider
```
I suggested changing the heading above to ##. If you agree, then this should be ###.